dvc.yaml

```diff
@@ -36,7 +36,7 @@ stages:
     - saved_models
   evaluate:
     cmd:
-    - sbatch -W ${slurm-eval} machines/slurm.neural-lam.sh --config_path ./data/config.yaml --eval val --load
+    - sbatch -W ${slurm-eval} --chdir ~/logs --export ALL,DVC_WORKING_DIR=$(pwd) machines/slurm.neural-lam.sh --config_path ./data/config.yaml --eval val --load
```

Comment:
@SimonKamuk is the `--export` needed? `MLLAM_VENV_PATH` is currently imported without this specific section. Do we need to add that too here?